src/UI/Handlebars/Helpers/MetaOption.js

```diff
@@ -0,0 +1,73 @@
+var Handlebars = require('handlebars');
+
+var metaOptionCurrentConfig = {};
+
+metaOptionCurrentConfig = require('../../../../_output/UI/Content/metaOption.json');
```

Comment:
This is an ugly part.. Still don't know, how to make this right..

see maxkoryukov/sonarr#1